### PR TITLE
Replacing outdated checklist & link. #1457

### PR DESCRIPTION
**Pull request by joelbyford**
Appears that the CSF checklist referenced in the document is no longer published or available.  Adding a new link to Azure docs page for similar blueprint documentation which is now using Azure Policy to enforce.

### CODE REVIEW

**Comment by mypil** [comment]
JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/offering-nist-csf.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ NIST Cybersecurity Framework (CSF) is a voluntary Framework that consists of sta
 
 Learn how to accelerate your NIST Cybersecurity Framework deployment with Compliance Score and our Azure Security and Compliance Blueprint:
 
-- [Download the Azure Security and Compliance Blueprint - NIST CSF Risk Assessment Checklist](https://servicetrust.microsoft.com/ViewPage/Blueprint?command=Download&downloadType=Document&downloadId=11e61716-11ab-4b11-9a9a-434b33a62ff9&docTab=fc060920-cdb8-11e7-bacf-0bf52b09d912_NIST_CSF_Blueprint)
+- [Overview of the NIST SP 800-53 R4 blueprint sample](https://docs.microsoft.com/en-us/azure/governance/blueprints/samples/nist-sp-800-53-rev4/)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/offering-nist-csf.md]:
```suggestion
- [Overview of the NIST SP 800-53 R4 blueprint sample](https://docs.microsoft.com/azure/governance/blueprints/samples/nist-sp-800-53-rev4/)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Please remove the locale from the link address.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@robmazz - PR has been copy edited and is ready for final review. 

Thank you.

**Pull request merged**